SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -408,6 +421,125 @@ void FSpatialGDKEditorToolbarModule::OnPropertyChanged(UObject* ObjectBeingModif
+	Writer->WriteObjectStart();
+	Writer->WriteValue(TEXT("name"), TEXT("bridge_qos_max_timeout"));
+	Writer->WriteValue(TEXT("value"), TEXT("0"));
+	Writer->WriteObjectEnd();
+	Writer->WriteObjectStart();
+	Writer->WriteValue(TEXT("name"), TEXT("bridge_soft_handover_enabled"));
+	Writer->WriteValue(TEXT("value"), TEXT("false"));
+	Writer->WriteObjectEnd();
+	Writer->WriteObjectStart();
+	Writer->WriteValue(TEXT("name"), TEXT("qos_max_unacked_pings_rate"));
+	Writer->WriteValue(TEXT("value"), TEXT("10"));
+	Writer->WriteObjectEnd();
+	Writer->WriteArrayEnd();
+	Writer->WriteObjectStart(TEXT("snapshots"));
+	Writer->WriteValue(TEXT("snapshot_write_period_seconds"), 0);
+	Writer->WriteObjectEnd();
+	Writer->WriteObjectEnd(); // World section end
+	Writer->WriteObjectStart(TEXT("load_balancing")); // Load balancing section begin
+	Writer->WriteArrayStart("layer_configurations");
+	Writer->WriteObjectStart();
+	Writer->WriteValue(TEXT("layer"), TEXT("UnrealWorker"));
+	Writer->WriteObjectStart("rectangle_grid");
+
+	int Cols = 1;
+	int Rows = 1;
+
+	if (const ULevelEditorPlaySettings* PlayInSettings = GetDefault<ULevelEditorPlaySettings>())
+	{
+		int NumServers = 1;
+		PlayInSettings->GetPlayNumberOfServers(NumServers);
+		
+		if (NumServers <= 2)
+		{
+			Cols = NumServers;
+			Rows = 1;
+		}
+		else
+		{
+			//Find greatest divisor.
+			int GreatestDivisor = 1;
+			for (int Divisor = 2; Divisor <= FMath::Sqrt(NumServers); Divisor++)
+			{
+				if (NumServers % Divisor == 0)
+				{
+					GreatestDivisor = NumServers / Divisor;
+				}
+			}
+
+			Cols = NumServers / GreatestDivisor;
+			Rows = GreatestDivisor;
+		}
+	}
+
+	Writer->WriteValue(TEXT("cols"), Cols);
+	Writer->WriteValue(TEXT("rows"), Rows);
+	Writer->WriteObjectEnd();
+	Writer->WriteObjectStart(TEXT("options"));
+	Writer->WriteValue(TEXT("manual_worker_connection_only"), true);
+	Writer->WriteObjectEnd();
+	Writer->WriteObjectEnd();
+	Writer->WriteArrayEnd();
+	Writer->WriteObjectEnd(); // Load balancing section end
+	Writer->WriteArrayStart(TEXT("workers")); // Workers section begin
+	Writer->WriteObjectStart();
+	Writer->WriteValue(TEXT("worker_type"), TEXT("UnrealWorker"));
+	Writer->WriteRawJSONValue("flags",  TEXT("[]"));
+	Writer->WriteArrayStart("permissions");
+	Writer->WriteObjectStart();
+	Writer->WriteObjectStart(TEXT("all"));
+	Writer->WriteObjectEnd();
+	Writer->WriteObjectEnd();
+	Writer->WriteArrayEnd();
+	Writer->WriteObjectEnd();
+	Writer->WriteObjectStart();
+	Writer->WriteValue(TEXT("worker_type"), TEXT("UnrealClient"));
+	Writer->WriteRawJSONValue("flags", TEXT("[]"));
+	Writer->WriteArrayStart("permissions");
+	Writer->WriteObjectStart();
+	Writer->WriteObjectStart(TEXT("all"));
+	Writer->WriteObjectEnd();
+	Writer->WriteObjectEnd();
+	Writer->WriteArrayEnd();
+	Writer->WriteObjectEnd();
+	Writer->WriteArrayEnd(); // Worker section end
+	Writer->WriteObjectEnd(); // End of json
+
+	Writer->Close();
+
+	if (FFileHelper::SaveStringToFile(Text, *LaunchConfigPath))
```

Comment:
Could we just check if this is false (and if so return false), otherwise return true at the bottom of the func?